Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs

```diff
@@ -44,6 +44,7 @@ public void SetUp ()
     }
 
     [Test]
+    [Ignore ("Test does not work properly after Firefox browser upgrade (Version 89).")]
```

Comment:
Hmm... that's actually just firefox, isn't it? so we need to do this ignore conditional for only-firefox.